frontend/demo/src/pages/Inbox/Messenger/MessengerListItem/index.tsx

```diff
@@ -21,7 +21,7 @@ const MessengerListItem = (props: MessengerListItemProps) => {
     return conversation && <AvatarImage contact={conversation.contact} />;
   };
 
-  const messageText = message.content[0].text;
+  const messageText = message.content;
```

Comment:
So this messageText will be a stringified JSON of the form
```{\"text\":\"Hello World\"}```

Not sure if the goal is to display just that, because if not then I think it would make sense to use @thorstenairy's render library here.

Reply:
@chrismatix yeah @thorstenairy is parsing the messages in his PR: I'll rebase when Thorsten merges his PR to use his changes